doc/source/whatsnew/v1.2.0.rst

```diff
@@ -225,33 +225,32 @@ to use this feature.
 
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
-- Added ``day_of_week``(compatibility alias ``dayofweek``) property to ``Timestamp``, ``DatetimeIndex``, ``Period``, ``PeriodIndex`` (:issue:`9605`)
-- Added ``day_of_year`` (compatibility alias ``dayofyear``) property to ``Timestamp``, ``DatetimeIndex``, ``Period``, ``PeriodIndex`` (:issue:`9605`)
-- Added :meth:`~DataFrame.set_flags` for setting table-wide flags on a ``Series`` or ``DataFrame`` (:issue:`28394`)
+- Added ``day_of_week`` (compatibility alias ``dayofweek``) property to :class:`Timestamp`, :class:`DatetimeIndex`, :class:`Period`, :class:`PeriodIndex` (:issue:`9605`)
+- Added ``day_of_year`` (compatibility alias ``dayofyear``) property to :class:`Timestamp`, :class:`DatetimeIndex`, :class:`Period`, :class:`PeriodIndex` (:issue:`9605`)
+- Added :meth:`~DataFrame.set_flags` for setting table-wide flags on a :class:`Series` or :class:`DataFrame` (:issue:`28394`)
```

Comment:
Personally, I don't find it needed to make each occurence of Series and DataFrame into a link: 1) those pages are not that useful and 2) those would get linked *many* times. 
In this specific sentence, the relevant link is the one to `set_flags`. By having this the only link in the sentence, it stands out more.

Reply:
Makes sense. To restate as a general rule: unless Series/DataFrame are being directly changed, do not link.